python/hail/expr/column.py

```diff
@@ -0,0 +1,1192 @@
+from __future__ import print_function  # Python 2 and 3 print compatibility
+
+from hail.java import *
+from hail.htypes import *
+from hail.representation import *
+
+
+def to_expr(arg):
+    if isinstance(arg, Column):
+        return arg.expr
+    elif isinstance(arg, str):
+        return "\"" + arg + "\""
+    elif isinstance(arg, bool):
+        return "true" if arg else "false"
+    elif isinstance(arg, long):
+        return "{}.toInt64()".format(arg)
+    elif isinstance(arg, list):
+        return "[" + ", ".join([to_expr(a) for a in arg]) + "]"
+    elif isinstance(arg, set):
+        return "let xfsjd = [" + ", ".join([to_expr(a) for a in arg]) + "] in xfsjd.toSet()"
+    elif isinstance(arg, Variant):
+        return "Variant(\"" + str(arg) + "\")"
+    elif isinstance(arg, Locus):
+        return "Locus(\"" + str(arg) + "\")"
+    elif isinstance(arg, Interval):
+        return "Interval(\"" + str(arg) + "\")"
+    elif isinstance(arg, Struct):
+        return "{" + ", ".join([k + ":" + to_expr(v) for k, v in arg._attrs.iteritems()]) + "}"  # FIXME: Struct constructor should take kwargs (ordered in Python 3.6)
+    elif callable(arg):
+        return arg
+    else:
+        return str(arg)
```

Comment:
Do you know what might go into this case?  For a case this long and complicated, I'd probably prefer an error here so we know explicit what we're missing.

Reply:
Same with other big case analyses.